.github/workflows/vita-ci.yml

```diff
@@ -0,0 +1,152 @@
+name: Vita CI
+
+on:
+  push:
+    branches:
+      - master
+      - devel
+      - wip-vita-support
+      - wip-vita-master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: ${{ matrix.config.name }} | ${{ matrix.config.build_type }}
+    runs-on: ${{ matrix.config.os }}
+    env:
+        VITASDK: /usr/local/vitasdk
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {
+            name: "Vita build",
+            os: ubuntu-20.04,
+            extra_options: "-DTHEXTECH_EXECUTABLE_NAME=thextech -DCMAKE_TOOLCHAIN_FILE=/usr/local/vitasdk/share/vita.toolchain.cmake -DVITA_APP_NAME=\"TheXTech Vita\" -DVITA_TITLEID=\"THEXTECH0\" -DVITA_VERSION=\"01.05\"",
+            deps_cmdline: "sudo apt-get update -qq && sudo apt-get install -qq cmake ninja-build cmake ninja-build lftp git-core python zip",
+            generator: "Unix Makefiles",
+            build_type: "MinSizeRel",
+            executable_name: "thextech",
+            assets_url: "https://wohlsoft.ru/TheXTech/_downloads/thextech-adventure-of-demo-assets-full.7z",
+            subdir_name: "thextech-vita",
+            upload_directory: "www/vita/"
+          }
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set output
+      id: vars
+      run: echo ::set-output name=short_ref::${GITHUB_REF#refs/*/}
+
+    - name: Install Dependencies
+      shell: bash
+      run: |
+        if [[ ! -z "${{ matrix.config.deps_cmdline }}" ]]; then
+          eval ${{ matrix.config.deps_cmdline }}
+        fi
+        # Pull submodules
+        git submodule init
+        git submodule update --recursive --remote
+
+    - name: Install Vita SDK
+      run: |
+        cd ..
+        export PATH=$VITASDK/bin:$PATH
+        git clone https://github.com/vitasdk/vdpm
+        cd vdpm
+        ./bootstrap-vitasdk.sh
+        ./install-all.sh
+
+# NOTE: 2/13/2022 Current Vita version does not use cglm or vitaGL renderer. All is handled by SDL2 at the moment.
```

Comment:
Right. Whenever you add the new renderers, feel free to tweak the boundary between render_base and render_SDL as you see fit (since this will probably be our first non-SDL renderer to make it into the main branch)